main/AgentDocs.h

```diff
@@ -26,7 +26,7 @@ class AgentDocs {
  public:
   void generate(const std::string &docsdir, std::ostream &genStream);
  private:
-  [[nodiscard]] inline std::string extractClassName(const std::string &processor) const;
+  [[nodiscard]] static inline std::string extractClassName(const std::string &processor);
```

Comment:
this isn't really `inline`

Reply:
Updated in d3b1db61b23d80b28385fde5f77af73c4661b95a